storage/Devices/DiskImpl.cc

```diff
@@ -46,18 +42,19 @@ namespace storage
     {
 	vector<string> ret;
 
-	for (const string& name : systeminfo.getDir(SYSFSDIR "/block"))
+	for (const string& short_name : systeminfo.getDir(SYSFSDIR "/block"))
 	{
-	    // we do not treat mds as disks although they can be partitioned since kernel 2.6.28
-	    if (boost::starts_with(name, "md") || boost::starts_with(name, "loop"))
+	    string name = "/dev/" + short_name;
+
+	    if (Md::Impl::is_valid_name(name) || boost::starts_with(name, "/dev/loop"))
```

Comment:
Yes, when a Loop class gets added there will be a is_valid_name function like with Md. BUt so far that class does not exist.